### PR TITLE
[TACHYON-734] Resolve javadoc warnings in server and minicluster

### PR DESCRIPTION
**Pull request by miasantreble**
https://tachyon.atlassian.net/browse/TACHYON-734

### CODE REVIEW

File: servers/src/main/java/tachyon/worker/block/BlockMetadataManagerView.java [hunk 1]
```diff
@@ -55,9 +55,8 @@
    * TODO: incrementally update the view
    *
    * @param manager which the view should be constructed from
-   * @param pinnedInodes, a set of pinned nodes
-   * @param lockedBlocks, a set of locked blocks
-   * @return BlockMetadataManagerView constructed
+   * @param pinnedInodes a set of pinned nodes
```
Comment (gpang) [inline servers/src/main/java/tachyon/worker/block/BlockMetadataManagerView.java]: "nodes" -> "inodes"

**Comment by gpang** [comment]
I just left a minor comment. Thanks for the PR!

File: servers/src/main/java/tachyon/worker/block/BlockMetadataManagerView.java (hunk 1 above)
Comment (miasantreble) [inline servers/src/main/java/tachyon/worker/block/BlockMetadataManagerView.java]: done

**Comment by apc999** [comment]
+1 LGTM

**Comment by gpang** [comment]
LGTM

**Pull request merged**